scripts/start_avd.py

```diff
@@ -0,0 +1,102 @@
+import sys
+import os
+import platform
+import subprocess
+import django
+from os.path import dirname, abspath
+
+MobSF_path = dirname(dirname(abspath(__file__)))
+sys.path.append(MobSF_path)
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'MobSF.settings')
+django.setup()
```

Comment:
Why these code?
From line 8-11 ?

Reply:
So I could use the Django settings interface (instead of parsing settings.py manually or passing arguments into the script)
It basically "starting" the Django engine of the project, just so I could use the setting